client/src/components/ErrorMessage.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState, useEffect } from 'react';
+import { Callout } from 'plaid-threads/Callout';
+import { IconButton } from 'plaid-threads/IconButton';
+import { CloseS2 } from 'plaid-threads/Icons/CloseS2';
+
+import useErrors from '../services/errors';
+
+//  Allows user to input their personal assets such as a house or car.
+
+export default function ErrorMessage() {
+  const [show, setShow] = useState(false);
+  const [message, setMessage] = useState('');
+  const { error, resetError } = useErrors();
+
+  useEffect(() => {
+    const institution = error.institution;
```

Comment:
If you want to keep your specific error state messages, we should at least include something like the above for the `default` case.

Reply:
These error messages are for the end-user and not developers, so I won't have them check the error documentation.  I would like to keep the specific messages, but I will add a default message as well.

Reply:
actually, i can use the display_message in the error object!  perfect!!!

Reply:
I also looked at using the `display_message`, but some of them were null.  I think I like being able to respond to end-users with specific messages on individual errors.  but i will add a `default` case.

Reply:
I guess the question is, since mostly developers will be using Pattern, would we always want to show the actual `error_code` since that's what they'd want to see?

Or if you just want to use user friendly messages, why not just use `display_message` and if `null`, use your default case? So no hard coding needed.